app/helpers/vita_min_form_builder.rb

```diff
@@ -63,16 +65,19 @@ def vita_min_state_file_select(
       method,
       label_contents(
         label_text,
-        options[:help_text],
+        has_help_text: !!options[:help_text],
         optional: options[:optional],
         ),
       class: label_class,
       )
     html_options_with_errors = html_options.merge(error_attributes(method: method))
 
+    help_html = help_text_html(options[:help_text], :method)
```

Comment:
shouldn't we be using the method variable?
```suggestion
    help_html = help_text_html(options[:help_text], method)
```